spec/models/feedbook/answer_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Feedbook::Answer, type: :model do
+  it { should belong_to(:question) }
```

Comment:
This kind of tests are a bit obvious, at least to me, because you didn't change anything from the logic of ActiveRecord, which is supposed to have tested how `belongs_to` work. It's not bad, but keep in mind that

Reply:
You don't need to test what has already been tested 😬